fitnessllm_dataplatform/stream/strava/services/api_interface.py

```
@@ -0,0 +1,141 @@
+        )
+        logger.info("Refreshed strava access token")
+
+    def instantiate_strava_lib(self, strava_access_dict_from_redis: dict):
+        """Instantiate strava client."""
+        strava_access_token = (
+            strava_access_dict_from_redis["access_token"]
+            if strava_access_dict_from_redis
+            else None
+        )
+        if not strava_access_token:
+            return None
```

Ruff (0.8.2)

Do not explicitly `return None` in function if it is the only possible return value

Remove explicit `return None`

(RET501)